tests/Stripe/TransferReversalTest.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Stripe;
+
+class TransferreversalTest extends TestCase
```

Comment:
`Transferreversal` should be `TransferReversal`.

Reply:
Fixed too